packages/dependency-profiles/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@fluentui-react-native/dependency-profiles",
+  "version": "0.0.0",
```

Comment:
Should this start at a non-zero number?

Reply:
Good question. @rurikoaraki do you know? I think I've seen new packages start at `0.0.0` before but also `0.1.0`.

Reply:
Either is fine. Beachball will bump it to a reasonable number as soon as it gets a chance to run